starter-code/src/functions-and-arrays.js

```diff
@@ -1,31 +1,74 @@
 // Iteration #1: Find the maximum
+const maxOfTwoNumbers = (num1, num2) => {
+  if (num1 < num2) {
+    return num2}
+  else {
+   return num1;
+  }
+ }
+ //Iteration #2: Find longest word
+ const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-// Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+ const findLongestWord = (arr) => {
```

Comment:
En esta no estaban devolviendo correctamente el null, por que la variable let comienza vacia, primero tienen que poner la condicional y si esta no se cumple devolver un null. cambiando el orden funciona.

`const findLongestWord = arr => {`
  `if (arr.length === 0) return null`
 ` let size = arr[0].length`
 ` let position = 0`
`  for (let i = 0; i < arr.length; i++) {`
  `  if (size < arr[i].length) {`
   `   position = i`
   `   size = arr[i].length`
   ` }`
 ` }`
  `return arr[position]`
`}`